lnd_grpc/lnd_grpc.py

```diff
@@ -93,26 +98,27 @@ def macaroon(self):
         try:
             with open(self.macaroon_path, 'rb') as f:
                 macaroon_bytes = f.read()
-                self._macaroon = codecs.encode(macaroon_bytes, 'hex')
-                return self._macaroon
+                macaroon = codecs.encode(macaroon_bytes, 'hex')
```

Comment:
This variable was not being used anywhere else in the class, so I made it a local instead of a private attribute